cocos/platform/android/CCGLViewImpl-android.cpp

```diff
@@ -51,7 +51,7 @@ void initExtensions() {
 
 NS_CC_BEGIN
 
-GLViewImpl* GLViewImpl::createWithRect(const std::string& viewName, Rect rect, float frameZoomFactor)
+GLViewImpl* GLViewImpl::createWithRect(const std::string& viewName, Rect rect, float frameZoomFactor, bool _ignored_resizable)
```

Comment:
The default parameter are banned in `Cocos Coding Style`: https://github.com/cocos2d/cocos2d-x/wiki/Cpp--Coding-Style#default-arguments